src/Qwiq.Core.Rest/Project.cs

```diff
@@ -64,7 +66,27 @@ internal Project([NotNull] TeamProjectReference project, [NotNull] WorkItemStore
                                               .GetResult();
 
                             return new Node(result).ChildNodes;
-                        }))
+                        }),
+                new Lazy<IQueryFolderCollection>(() =>
+                {
+                    return new QueryFolderCollection(() =>
+                    {
+                        //BUGBUG: There is a bug in the GetQueryAsync in vsts where if a folder contains a '+' character it will return 404, even if the folder exists see here: https://developercommunity.visualstudio.com/content/problem/123660/when-a-saved-query-folder-contains-a-character-it.html
+                        QueryHierarchyItem FolderExpansionFunc(string path)
+                        {
+                            try
+                            {
+                                return store.NativeWorkItemStore.Value.GetQueryAsync(project.Id, path, QueryExpand.Wiql, 2).Result;
```

Comment:
Should `2` be turned into a constant to be self-documenting?

Reply:
Yup, will do.

Reply:
updated